### PR TITLE
[test-only] Check that image editor open

### PR DESCRIPTION
**Pull request by ScharfViktor**
changed test in master branch for testing: https://github.com/owncloud/ocis/pull/8608

### CODE REVIEW

**Comment by ScharfViktor** [comment]
@kobergj if you want to check tests with your changes -> just update `ocis commitID`

**Comment by ScharfViktor** [comment]
image

issue still relevant in CI. I cannot get why. localy works correct

**Comment by ScharfViktor** [comment]
I figured out what is issue. step `uploads the following local file(s) into personal space using API` doesn't work correct. 
looking into the problem..

**Review by saw-jan — APPROVED** [review]
:+1:
we also want to have it in stable-8 once the fix is ported to the ocis stable-5

**Comment by saw-jan** [comment]
> we also want to have it in stable-8 once the fix is ported to the ocis stable-5

I see the PR: https://github.com/owncloud/web/pull/10565 :+1:

File: tests/e2e/cucumber/features/smoke/shares/link.feature [hunk 1]
```diff
@@ -169,10 +170,35 @@ Feature: link
     And "Carol" closes the file viewer
     When "Carol" opens the public link "imageLink"
     And "Carol" unlocks the public link with password "%public%"
+    # https://github.com/owncloud/ocis/issues/8602
```
Comment (saw-jan) [inline tests/e2e/cucumber/features/smoke/shares/link.feature:173]: @ScharfViktor I resolved the conflict and looks good from my side.

File: tests/e2e/cucumber/features/smoke/shares/link.feature [hunk 2]
```diff
@@ -86,10 +86,10 @@ Feature: link
...
       | folderPublic/shareToBrian.txt | some text |
       | folderPublic/shareToBrian.md  | readme    |
-    And "Alice" uploads the following local file into personal space using API
-      | localFile                     | to             |
-      | filesForUpload/simple.pdf     | simple.pdf     |
-      | filesForUpload/testavatar.jpg | testavatar.jpg |
+    And "Alice" uploads the following resource
+      | resource       |
+      | simple.pdf     |
+      | testavatar.jpg |
```
Comment (saw-jan) [inline tests/e2e/cucumber/features/smoke/shares/link.feature:92]: .

**Pull request merged**